### DISCUSSION TITLE
How do I implement "Go to selected step"?

### DISCUSSION THREAD

**Discussion by sanadianpking** [gilbarbara/react-joyride#623] [Q&A] [answered]
How do I implement "Go to selected step" from the init screen by selecting the step number?

**Comment by gilbarbara** [comment, marked as answer]
hey @sanadianpking 

You can use the `stepIndex` prop to control the tour. Check the [controlled](https://github.com/gilbarbara/react-joyride-demo/blob/master/src/Controlled/index.tsx#L179) demo for a detailed example.
